### ISSUE TITLE
Pretrained Model Download Issues

### ISSUE THREAD

**Issue by hung-z** [jacobswan1/SEED#2]
Thanks for the great work on the Self-supervised Distillation For Visual Representation paper.
I have some trouble with the code.
I cannot download the pre-trained seed model and teacher model using the link in the GitHub repository. Can you check for us about that? We really need the seed model to analyze.
The issues is that when I click to the download link, it showed that,
This XML file does not appear to have any style information associated with it. The document tree is shown below.

![Screenshot from 2021-11-02 11-13-44](https://user-images.githubusercontent.com/62822078/139796327-d535ecf7-c74d-4e75-9d56-779b71f01ca9.png)

**Comment by jacobswan1** [comment]
Hi hung, I have just updated the download link. Checkpoints should be good to be downloaded now.
